dns/dyndns/src/etc/inc/plugins.inc.d/dyndns/phpDynDNS.inc

```diff
@@ -705,41 +709,52 @@ class updatedns
                 curl_setopt($ch, CURLOPT_URL, $server);
                 break;
             case 'cloudflare':
-                $dnsServer ='api.cloudflare.com';
-                $dnsHost = str_replace(' ', '', $this->_dnsHost);
-                $host_names = explode('.', $dnsHost);
-                $bottom_host_name = $host_names[count($host_names) - 2] . '.' . $host_names[count($host_names) - 1];
+            case 'cloudflare-v6':
+                $baseUrl = 'https://api.cloudflare.com/client/v4';
```

Comment:
v4 is cloudflare versioning, not IPv4, right?

Reply:
That's correct

Reply:
ok, thanks. should have read on, the url was moved, not added so sorry for the noise :)